lib/firmata.js

```diff
@@ -545,6 +545,17 @@ var Board = function(port, options, callback) {
         this.setSamplingInterval(settings.samplingInterval);
       }
       if (settings.skipCapabilities) {
+        this.analogPins = this.settings.analogPins || this.analogPins;
+        this.pins = this.settings.pins || this.pins;
```

Comment:
I really hate that `settings` became a property of the instance. Can you kill that and just do `settings.foo`? Eventually I want to eliminate `this.settings`